euphonic/broadening.py

```diff
@@ -1,27 +1,30 @@
 """
 Functions for broadening spectra
 """
-from typing import Callable, Union
+from typing import Callable, Literal, Union
 import warnings
 
 import numpy as np
 from numpy.polynomial import Chebyshev
+from pint import Quantity
 from scipy.optimize import nnls
 from scipy.stats import norm
 from scipy.signal import convolve
 
-from euphonic import ureg, Quantity
+from euphonic import ureg
 
 
-def variable_width_broadening(bins: Quantity,
-                              x: Quantity,
-                              width_function: Callable[[Quantity], Quantity],
-                              weights: Union[np.ndarray, Quantity],
-                              width_lower_limit: Quantity = None,
-                              width_convention: str = 'fwhm',
-                              adaptive_error: float = 1e-2,
-                              shape: str = 'gauss',
-                              fit: str = 'cheby-log') -> Quantity:
+def variable_width_broadening(
+    bins: Quantity,
+    x: Quantity,
+    width_function: Callable[[Quantity], Quantity],
+    weights: Union[np.ndarray, Quantity],
+    width_lower_limit: Quantity = None,
+    width_convention: Literal['FWHM', 'STD'] = 'FWHM',
```

Comment:
I think the innermost function actually casts them to lower() so this works. I agree the inconsistency is annoying. Maybe it can quietly be changed without breaking anything, by strategically inserting .lower() in a few places...

It would be neat to use an enum that makes the upper- and lower-case versions valid and literally equivalent for comparison purposes, but it seems a bit cumbersome and overkill.

Reply:
Ok, for consistency I have made all the internal references and type-hints lower-case `'fwhm'`, `'std'`. It is still case-insensitive and the tests are unchanged, so this is a non-breaking change.